ui/threejs/moveToFit.js

```diff
@@ -16,7 +16,7 @@ export const moveToFit = ({
   scene,
   fitOffset = 1.2,
   withGrid = false,
-  gridLayer = SKETCH_LAYER,
+  gridLayer = GEOMETRY_LAYER,
```

Comment:
What does it mean that the grid is on the geometry layer now?

Reply:
It means that it won't be drawn over the top of geometry.